samples/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -233,6 +247,41 @@ static bool event_handler(const struct event_header *eh)
 		return false;
 	}
 
+	if (IS_ENABLED(CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE)) {
+		if (is_config_forward_event(eh)) {
+			struct config_forward_event *event = cast_config_forward_event(eh);
```

Comment:
`const struct...`

Reply:
unless ble api demands otherwise... I know they are not always marking cost things properly

Reply:
Fixed